apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -215,6 +227,20 @@ const NewPublication: FC<NewPublicationProps> = ({ publication }) => {
     onError
   });
 
+  const [broadcastDataAvailability] = useBroadcastDataAvailabilityMutation({
+    onCompleted: (data) => {
+      onCompleted();
+      if (data?.broadcastDataAvailability.__typename === 'RelayError') {
+        return toast.error(Errors.SomethingWentWrong);
+      }
+
+      if (data?.broadcastDataAvailability.__typename === 'CreateDataAvailabilityPublicationResult') {
+        push(`/posts/${data?.broadcastDataAvailability.id}`);
+      }
```

Comment:
Can we make an optimistic push to the feed?

Reply:
We can make it happen in next iteration